packages/material-ui/test/integration/MenuList.test.js

```diff
@@ -15,210 +16,170 @@ function FocusOnMountMenuItem(props) {
   return <MenuItem {...props} ref={listItemRef} tabIndex={0} />;
 }
 
-function TrackRenderCountMenuItem({ actions, ...other }) {
-  const renderCountRef = React.useRef(0);
+function TrackCommitCountMenuItem({ actions, ...other }) {
```

Comment:
So excited about a stable Profiler component where we can get rid of these workarounds.

Reply:
Get rid of it?!! I'm rather emotionally attached to that bit of code after all the time I invested into figuring out how to verify that it wasn't re-rendering on focus changes. :)

Reply:
Well this particular code is already nice because you went with commit tracking instead of the render tracking. That's already future proof when we run fuzzy tests in concurrent mode. Most of the tests around this issue go with render counting which will become unstable.